crates/op-rbuilder/src/tests/framework/op.rs

```diff
@@ -155,6 +167,16 @@ impl Service for OpRbuilderConfig {
                 .arg(flashbots_block_time.to_string());
         }
 
+        if let Some(max_da_tx_size) = self.max_da_tx_size {
+            cmd.arg("--builder.max-da-tx-size")
```

Comment:
Is this a flag of the builder? I though it is the batcher the one that updates these values.

Reply:
It'll need to be on the RPC, as it changes dynamically when the batcher is congested.

If you ensure the DA Config is shared between the payload builder and the OPAddOnsBuilder it should work ([ref](https://github.com/flashbots/op-rbuilder/pull/8/files#diff-a444c4b32610ee0cee1af7d1ae5608e877cca94ecb7e4b518c38bd80951f2e40R59)) out of the box.

We're running the two closed PR's on Base Sepolia (and briefly Base Mainnet) last week and it successfully was able to throttle on both networks.